### PR TITLE
feat: contract bindings

### PR DESCRIPTION
**Pull request by thomas192**
- [x] issue #33 
- [x] follows contribution [guide](https://github.com/keep-starknet-strange/blobstream-starknet/blob/main/CONTRIBUTING.md)
- [ ] code change includes tests
- [ ] breaking change

Created the bindings using Cainome.

I suppose next step would be testing, but Sepolia deployment is required if I'm not mistaken.

First time doing this, let me know if changes are needed.

### CODE REVIEW

**Review by drspacemn — COMMENTED** [review]
Is the `library` name autogenerated by cainome? Can we change it to `bindings`?

Also can you provide a line in the README.md for generating/updating the bindings?

File: library/.gitignore
Comment (drspacemn) [inline library/.gitignore:1]: we can delete and handle w/ the top-level `.gitignore`

**Comment by thomas192** [comment]
> Is the `library` name autogenerated by cainome? Can we change it to `bindings`?
> 
> Also can you provide a line in the README.md for generating/updating the bindings?

Indeed it can be named `bindings`.
Made the update.

**Comment by drspacemn** [comment]
brilliant. lgtm.

**Pull request merged**